Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -311,7 +301,7 @@ public virtual void RemoveRole(Player player)
         {
             TrackedPlayers.Remove(player);
             if (Scale != Vector3.zero)
```

Comment:
If you set this to Vector3.zero (0, 0, 0) it will always be true even when the player is in his normal size of Vector3.one (1, 1, 1)